code/modules/mob/living/carbon/human/human.dm

```diff
@@ -118,6 +118,19 @@
 /mob/living/carbon/human/machine/Initialize(mapload)
 	..(mapload, "Machine")
 
+/mob/living/carbon/human/machine/created
+	name = "automated machine-frame"
+
+/mob/living/carbon/human/machine/created/Initialize(mapload)
+	..()
+	real_name = "automated machine-frame ([rand(1, 1000)])"

```

Comment:
Could probably have a simple namelist; would be far fancier than just a number.

Alternatively, the ability to rename yourself when your brain is inserted into the thing? That might be abusable, though.

Reply:
The name is to accentuate that they're much like golems; generalized workforce automatons that have no self-determination beyond what their master gives them--it also helps strongly differentiate them from any on-station synthetics and gives players, at glance the notion that "hey, these guys are someone's constructs".